src/resources/postcss/components/full/_top-bar.pcss

```diff
@@ -71,3 +71,23 @@
 		}
 	}
 }
+
+/* -----------------------------------------------------------------------------
+ * Theme Overrides - Twenty Twenty One
+ * ----------------------------------------------------------------------------- */
+.tribe-theme-twentytwentyone {
+	button.tribe-events-c-top-bar__datepicker-button:focus {
+		outline-offset: 0;
+	}
+
+	.tribe-events {
+		/* Higher level of specificity to avoid using !import and override the theme styles. */
+		a.tribe-events-c-top-bar__nav-link--next.tribe-events-c-top-bar__nav-link:focus {
+			background: transparent;
+		}
+
+		a.tribe-events-c-top-bar__nav-link--prev.tribe-events-c-top-bar__nav-link:focus {
+			background: transparent;
+		}
+	}
+}
```

Comment:
same thing here, move into the `.tribe-events` block above, like the other theme overrides